evap/staff/tests/test_views.py

```diff
@@ -247,3 +247,57 @@ def test_no_second_update(self):
         # We should get redirected back to the questionnaire index.
         self.assertEqual(page.status_code, 302)  # REDIRECT
         self.assertEqual(page.location, '/staff/questionnaire/')
+
+
+class TestSemesterRawDataExportView(ViewTest):
+    url = '/staff/semester/1/raw_export'
+    test_users = ['staff']
+
+    @classmethod
+    def setUpTestData(cls):
+        mommy.make(UserProfile, username='staff', groups=[Group.objects.get(name='Staff')])
+        cls.student_user = mommy.make(UserProfile, username='student')
+        cls.semester = mommy.make(Semester)
+        cls.course_type = mommy.make(CourseType)
+        cls.course1 = mommy.make(Course, type=cls.course_type, semester=cls.semester, participants=[cls.student_user],
+            voters=[cls.student_user], name_de="Veranstaltung 1")
+        cls.course2 = mommy.make(Course, type=cls.course_type, semester=cls.semester, participants=[cls.student_user],
+            name_de="Veranstaltung 2")
+        mommy.make(Contribution, course=cls.course1, responsible=True, can_edit=True, comment_visibility=Contribution.ALL_COMMENTS)
+        mommy.make(Contribution, course=cls.course2, responsible=True, can_edit=True, comment_visibility=Contribution.ALL_COMMENTS)
+
+    def test_view_downloads_csv_file(self):
+        response = self.app.get(self.url, user='staff')
+        degrees1 = ", ".join([degree.name for degree in self.course1.degrees.all()])
+        degrees2 = ", ".join([degree.name for degree in self.course2.degrees.all()])
+        expected_content = "Name;Degrees;Type;Single result;State;#Voters;#Participants;#Comments;Average degree\r\n" + \
+            self.course1.name_en + ";" + degrees1 + ";" + self.course_type.name_en + ";False;" + self.course1.state + ";" + \
+            str(self.course1.num_voters) + ";" + str(self.course1.num_participants) + ";" + str(self.course1.textanswer_set.count()) + ";\r\n" + \
+            self.course2.name_en + ";" + degrees2 + ";" + self.course_type.name_en + ";False;" + self.course2.state + ";" + \
+            str(self.course2.num_voters) + ";" + str(self.course2.num_participants) + ";" + str(self.course2.textanswer_set.count()) + ";\r\n"
```

Comment:
how about just putting the expected string here, so without accessing all the attributes, just the result? makes it less ugly and more strict